iggy/build.rs

```diff
@@ -0,0 +1,267 @@
+extern crate rmp_serde as rmps;
+extern crate serde;
+extern crate serde_derive;
+
+use std::error::Error;
+use std::fs::File;
+use std::io::Write;
+
+use errors_repository::PreprocessedErrorRepositoryEntry;
+
+use crate::data_repository::{DataRepository, SledDb};
+use crate::errors_repository::get_or_create;
+
+mod data_repository;
+mod errors_repository;
+
+fn snake_to_pascal_case(s: &str) -> String {
```

Comment:
if it fits you, just use this crate:
https://docs.rs/convert_case/latest/convert_case/enum.Case.html

less boilerplate code :)

Reply:
Changed it.